backend/src/routes/v2/model/file/deleteFile.ts

```diff
@@ -18,7 +19,11 @@ interface DeleteFileResponse {
 export const deleteFile = [
   bodyParser.json(),
   async (req: Request, res: Response<DeleteFileResponse>) => {
-    const _ = parse(req, deleteFileSchema)
```

Comment:
What does the 'const _' do and why is it not implemented anymore?

Reply:
It originally wasn't being used before as the response was just mocked. It's now been been re implemented a few lines below